blocks/api/helpers.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import { every, keys, isEqual } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getDefaultBlockName } from './registration';
+import { createBlock } from './factory';
+
+/**
+ * Determines whether the block is a default block
+ * and its attributes are equal to the default attributes
+ * which means the block is untouched.
+ *
+ * @param  {WPBlock} block Block Object
+ *
+ * @return {boolean}       Whether the block is an untouched default block
+ */
+export function isUntouchedDefaultBlock( block ) {
```

Comment:
Minor: What do you think of "Unmodified" instead of "Untouched" ? Not sure "touch" is the best way for describing modifications to a bock.